frontend/src/component/admin/groups/GroupForm/GroupForm.tsx

```diff
@@ -146,16 +192,50 @@ export const GroupForm: FC<IGroupForm> = ({
                     elseShow={() => (
                         <StyledDescriptionBlock>
                             <Box sx={{ display: 'flex' }}>
-                                You can enable SSO groups syncronization if
+                                You can enable SSO groups synchronization if
                                 needed
-                                <HelpIcon tooltip="SSO groups syncronization allows SSO groups to be mapped to Unleash groups, so that user group membership is properly synchronized." />
+                                <HelpIcon tooltip="SSO groups synchronization allows SSO groups to be mapped to Unleash groups, so that user group membership is properly synchronized." />
                             </Box>
                             <Link data-loading to={`/admin/auth`}>
                                 <span data-loading>View SSO configuration</span>
                             </Link>
                         </StyledDescriptionBlock>
                     )}
                 />
+                <ConditionallyRender
+                    condition={groupRootRolesEnabled}
+                    show={
+                        <>
+                            <StyledInputDescription>
+                                <Box sx={{ display: 'flex' }}>
+                                    Do you want to associate a root role with
+                                    this group?
+                                    <HelpIcon tooltip="When you associate an Admin or Editor role with this group, users in this group will automatically inherit the role globally. Note that groups with a root role association cannot be assigned to projects." />
+                                </Box>
+                            </StyledInputDescription>
+                            <StyledAutocompleteWrapper>
+                                <Autocomplete
+                                    data-testid={'test'}
```

Comment:
Perhaps use something a bit more specific and explicit?
```suggestion
                                    data-testid="GROUP_ROOT_ROLE"
```

Reply:
I like!